docs/develop/javascript-objects.md

```diff
@@ -102,7 +102,7 @@ function main(workbook: ExcelScript.Workbook) {
 }
 ```
 
-The [Work with dates](../resources/excel-samples.md#work-with-dates) section of the samples has more Date-related scripts.
+The [Work with dates](../resources/excel-samples.md#dates) section of the samples has more Date-related scripts.
```

Comment:
caps on date seems odd?

Reply:
That article section is about the JavaScript Date object. That said, I'm ambiguously referring to dates here, so I'll change it.